rxandroid/build.gradle

```diff
@@ -115,8 +115,8 @@ bintray {
         licenses = ['Apache-2.0']
         labels = ['rxjava', 'reactivex']
         websiteUrl = 'https://github.com/ReactiveX/RxAndroid/'
-        issueTrackerUrl = 'https://github.com/ReactiveX/RxAndroid/issues'
-        vcsUrl = 'https://github.com/ReactiveX/RxAndroid.git'
+        issueTrackerUrl = 'https://github.com/ReactiveX/RxAndroid/issues/'
```

Comment:
I thought the '~/issue' would be a folder, so I thought it would be redirected to '~/issues/'.
I thought the other url was the same. But I tried it, but it was not.
So, I'll fix and push again. Thanks